api/post.go

```diff
@@ -72,6 +72,8 @@ func createPost(c *Context, w http.ResponseWriter, r *http.Request) {
 		post.CreateAt = 0
 	}
 
+	c.App.SanitizeProps(post)
```

Comment:
There any reason for this or could we simplify with `post.SanitizeProps()`?

Reply:
No reason, just follow the existing format/pattern.